CODEOWNERS

```diff
@@ -1,3 +1,5 @@
 components/builder-* @chefsalim
 components/butterfly* @christophermaier
 components/sup @christophermaier
+components/builder-web @cnunciato
```

Comment:
I think this line would need to go above `components/builder-*` to ensure that `builder-web` changes tagged you; otherwise, _every_ builder change would tag Salim.